metricbeat/docs/modules/oracle.asciidoc

```diff
@@ -28,14 +26,35 @@ image::./images/metricbeat-oracle-overview.png[]
 [float]
 
 == Requirements
-Oracle database requires a special OCI connector to get connected to the database. You can find more information about it in the https://www.oracle.com/database/technologies/instant-client.html[official link of Oracle's webpage]. Oracle's webpage contains a comprehensive manual about installation methods and troubleshooting. A small description is included here but follow Oracle's official documentation if you find some issue. The OCI Instant Client is available with the Oracle Universal Installer, RPM file or ZIP file. Download links can be found https://www.oracle.com/database/technologies/instant-client/downloads.html[here]. The Oracle Technology Network License Agreement with Oracle must be accepted to download the library.
 
-The OCI library install few Client Shared Libraries that must be referenced on the machine where Metricbeat is installed. The environment variable `LD_LIBRARY_PATH` is used to set the reference required by the Metricbeat library. For example, let's say you have downloaded the library into your `Downloads` folder, you can follow the steps below to install the ZIP library on `/usr/lib` (assuming you have `wget` and a zip unpacker):
+*Oracle Database Connection Pre-requisites*
```

Comment:
Solution ( https://github.com/elastic/beats/issues/30949)  Says: 
`Connectivity to Oracle can be facilitated in two ways either by using official Oracle libraries or by using a JDBC driver. These two approaches must be clearly spelled out in the documentation, whether they are supported or not and what libraries and settings must be configured, and where. `

So, kindly include the approaches that are supported and what isn't.

Reply:
Right, thank you. We started [this discussion](https://github.com/elastic/beats/issues/30949#issuecomment-1110761183) over the [PRD for the task](https://github.com/elastic/beats/issues/30949) for the same.

As mentioned there, based on our analysis, it won't be feasible to facilitate the connection to Oracle using JDBC. Should we mention the same or does anyone from the team has any difference of opinion for it?

Reply:
It's updated as discussed over the call. Please re-review the PR at your convenience.